### PR TITLE
Using Rails Form Helpers

### PR DESCRIPTION
**Pull request by didymus707**
- Form helpers such as Form_tag and Form_for were used in building rails forms in this commit
- Also, a plain html form was also built too

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<%= form_for @user, url: {action: 'edit'} do |f| %>
```
Comment (shshamim63) [inline app/views/users/edit.html.erb]: When editing an entry the corresponding next action should be 'update'. But you have defined edit. That's why, when I try to entry it throws an error related to the path. To fix it 
```ruby
<%= form_for @user do |f| %>
```
Rails is smart enough to find the update path when in terms of the edit.

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## Status :recycle: Changes Required :1234: 
___
Hi @didymus707 :wave: 
Almost there to complete this project. But there are some issues that are blocking me to approve it right now. Let's get to them.
___
### :spades: Mandatory Changes :spades: 
___
1. It seems the readme is containing the basic info which is provided when creating a rails app. The info required for this project is not it. Please generate descriptive readme with the project setup and how to use it related info.
2. While I tried to update a info it throws error related to the path. If you need to perform an edit operation it should pass through two consecutive actions, edit and update. I can only see the path related to edit. Where is the update route and action? Please fix the route and add an action in the user controller related to update.
image
___
### Please fix the above-mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<h1>Create a New User Form</h1>
+<!--
+<form accept-charset="UTF-8" action="/users" method="post">
```
Comment (kobiyoyo) [inline app/views/users/new.html.erb:3]: Its better written this way `<form action=users_path method='post' accept-charset="UTF-8">`

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <label for="password_confirmation">Password Confirmation:</label>
+  <input type="password" id="password_confirmation" name="user[password_confirmation]"><br><br>
+  <input type="submit" value="Submit">
+</form> -->
+
+<%# <%= form_tag('/users') do %>
```
Comment (kobiyoyo) [inline app/views/users/new.html.erb:16]: Its better written this way `<%= form_tag(users_path) do |f|%>`

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team      
#### Great Job so far, but there are a few more things required. 
### General
- [x] Lets give this project a proper title in the readme file.
- [x]  I can't see any button to enable me edit `User`
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by mosesogwo — APPROVED** [review]
**Hello:clap:. Congratulations :+1:. Your project is approved.**

You can now merge this PR.

**Cheers :stars:**

### Status: Approved:heavy_check_mark:

**Pull request merged**